docs/articles/developer-info/Notes-Toward-NUnit-4.0.md

```diff
@@ -2,9 +2,11 @@
 
 This page describes plans for the next major releases of NUnit, focusing on major changes we expect to make.
 
+Also see [Towards NUnit 4](xref:towardsnunit4) for more details on the changes.
+
 ## Extensions
 
-With the release of nunit3-console 4.0, we intend to stop distributing the **teamcity-event-listener** bundled with with the console as a standard extension. We will continue to support the `--teamcity` option in the console runner when the extension is installed separately by the user.
+With the release of nunit3-console 4.0, we intend to stop distributing the **teamcity-event-listener** bundled with the console as a standard extension. We will continue to support the `--teamcity` option in the console runner when the extension is installed separately by the user.
```

Comment:
❓ Purely curiosity - have we had a discussion about whether to switch `nunit3-console` and use `nunit-console` or at least `nunit4-console`? Since we're making breaking changes, nunit3-conosole v4 seems like the worst of all worlds.

Reply:
@CharliePoole raised this in https://github.com/nunit/nunit-console/issues/1215 but there were no _discussions_.

Reply:
Thank you for the extra context! Yes, I think a quick chat about that is warranted.

Reply:
Unfortunately, the names in Nuget is the same as their id, so changing them is like creating a whole new package.  So, as long as nuget.org doesn't allow us to change that name, we're stuck with the names we have.

Another point here is that we don't need any nunit4-console, nor any nunit4-engine to run nunit4.  Nunit 3.15.4 does the job just fine. So that should probably be refined.